### ISSUE TITLE
Major security error.

### ISSUE THREAD

**Issue by Mangadania** [siacs/Conversations#2366]
Hey Dev and users.

Just found a major security problem with this app. Im using latest app from appstore and latest android on a samsung phone 

Problem: Encrypted logs are saved, images and files are also saved on your device and not deleted by the app at ANY time.

1: I have automatic message deletion activated. Sat at 24 hours. 

2: Every single time I end an conversation i choose clear history. 

So there should be NO images, messages or any other information on the phones storage after 24 hours or after you clear your history.

But that`s not the case at all. If I go to the following location on my phone.

sdcard > Conversations > logs
Then I can see a directory for each account on the app. If press you will find .txt files from encrypted chats..(Why are these stored in cleartext at all ??) Today is the 11 march. But I see logs from 7-8 of march. So the delete after 24 hours function does not work. 

sdcard > Converstations > Media

Here you find ALL the media, images and so on ever sent by your phone since converstaions was installed. Nothing gets deleted.

Solutions:

- DON`T save logs at all. Please add this to the settings ASAP-

- Make the cleaning functions in the app work. When a conversation have ended and been manually or automatically cleaned. It should not be on the phone, text, video, images etc. Should be GONE.

Since this major security isse have been allowed to be unoticed (if not the devs have choosen not to fix it). Then I have no trust in this being fixed ever. So what to do.

Practical solutions.

- Find a new xmpp app that have been tested and where all included functions work. (Long shot)

- Add automatic wiping of these files and directories. NOT just rm .-rf directory. But wipe. This is sensitive private information. Suggestions for apps that can do this are more than welcome.

Regards
A unhappy camper

**Comment by iNPUTmice** [comment]
> sdcard > Conversations > logs

Those are the 'Exported log'. You created those logs by hitting the 'Export Logs' button yourself. Conversations usually doesn't store logs in there. The 'Clear history' and 'delete messages after x hours' is only for the internal messages.

**Comment by Mangadania** [comment]
sdcard > Conversations > logs

This seems to make sense. Hit that export button only once. 

But this does not explain why the media isnt deleted every 24 hours or when you manually end and clean a chat for history. 

Found a solution since this app doesnt delete those files. 

1: Delete with app. CCcleaner or alike with a set time. 
2: Wipe free space at a certain time with "Complete clean" from app store. 

Havn`t found an app that can shred specific files at a certain time. (yet)

**Comment by iNPUTmice** [comment]
> But this does not explain why the media isnt deleted every 24 hours or when you manually end and clean a chat for history.

Currently this is expected behavior. I created a separate issue (#2367) to track a potential behavior change.
